blogposts/announcing-sourcegraph-3.15.md

```diff
@@ -0,0 +1,294 @@
+---
+title: "Sourcegraph 3.15: Campaigns beta launch, faster GitHub repository permissions, experimental AND/OR operators, and major observability improvements"
+author: Christina Forney
+publishDate: 2020-04-20T10:00-07:00
+tags: [
+  blog, Campaigns, GitHub, 
```

Comment:
I'd just keep this at `blog` for now until we have a list of defined and supported tags unless @aileenrose, are we surfacing these to search engines somehow?